setup.cfg

```
@@ -0,0 +1,49 @@
+[metadata]
+name = airtigrs
+description = Orchestration of TIGRLab dataflow infrastructure
+long_description = file:README.md
+keywords = neuroimaging, etl, automation, orchestration
+classifiers =
+	Intended Audience :: Science/Research
+	Topic :: Scientific/Engineering
+	Programming Language :: Python :: 3.7
+	Programming Language :: Python :: 3.8
+	Programming Language :: Python :: 3.9
+project_urls =
+	GitHub = https://github.com/tigrlab/airtigrs
```

Should the repository name be changed to `airtigrs` or the text here to `air-tigrs`?

```suggestion
	GitHub = https://github.com/tigrlab/airtigrs
```

---

Yupp good catch!